juniper_official/Kernel/check-delayed-unrefs-anomaly.rule

```diff
@@ -0,0 +1,70 @@
+/*
+ * This rule checks if there are any delayed unrefs and if they are greater than
+ * a particular threshol (sysctl net.rt_nh_max_delayed_unrefs).
+ *
+ * Transient spikes in delayed unref count is expected in certain catastrophic or
+ * major events and is not a cause for worry. But, if this situation persists for
+ * significant amount of time (say in minutes), then it can result in traffic loss.
+ */
+iceberg {
+    topic kernel.ifstate {
+        rule check-delayed-unrefs-anomaly {
+            sensor kernel-ifstate-sensor {
+                open-config {
+                    sensor-name /junos/kernel-ifstate;
+                    frequency 30s;
+                }
+            }
+            field delayed-unrefs-cnt {
+                sensor kernel-ifstate-sensor {
+                    path /junos/kernel-ifstate/delayed-unrefs-cnt;
+                }
+                type integer;
+                description "Delayed unrefs count";
+            }
+            field delayed-unrefs-max {
+                sensor kernel-ifstate-sensor {
+                    path /junos/kernel-ifstate/delayed-unrefs-max;
+                }
+                type integer;
+                description "Delayed unrefs max count";
+            }
+            trigger report-delayed-unrefs {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed